test/transport.test.js

```diff
@@ -15,6 +15,32 @@ const writer = require('flush-write-stream')
 const { pid } = process
 const hostname = os.hostname()
 
+async function installTransportModule () {
+  try {
+    await uninstallTransportModule()
+  } catch {}
+  try {
+    await symlink(
+      join(__dirname, 'fixtures', 'transport'),
+      join(__dirname, '..', 'node_modules', 'transport')
+    )
+  } catch (err) {
+    if (!isYarnPnp) {
+      throw err
+    }
+  }
+}
+
+async function uninstallTransportModule () {
+  try {
+    await unlink(join(__dirname, '..', 'node_modules', 'transport'))
+  } catch (err) {
+    if (!isYarnPnp) {
```

Comment:
it's better to avoid the unlink if `isYarnPnp`, i.e. return at the start of the function.